### PR TITLE
test merge of joq/unit_tests for Travis

### PR DESCRIPTION
**Pull request by jack-oquin**
xref: utexas-bwi/bwi#38

### CODE REVIEW

**Comment by jack-oquin** [comment]
I was expecting that failure. 

We can't declare our dependencies on the segway packages because they are not released.

**Comment by jack-oquin** [comment]
The last commit should also fail, but this time due to roslaunch check errors.

**Comment by jack-oquin** [comment]
This is working now, and can probably be merged after the IJCAI demo next Tuesday.

**Comment by jack-oquin** [comment]
@piyushk: please review

File: segbot_bringup/CMakeLists.txt [hunk 1]
```diff
@@ -23,17 +23,18 @@ install(PROGRAMS
...
   roslaunch_add_file_check(launch/segbot_v2.launch)
+  #roslaunch_add_file_check(launch/segbot_v3.launch)
+  roslaunch_add_file_check(launch/segway_base.launch)
+  roslaunch_add_file_check(launch/includes)
   roslaunch_add_file_check(launch/includes/auxiliary.segbot_v1.hokuyo_only.launch.xml)
+  roslaunch_add_file_check(launch/includes/auxiliary.segbot_v1.launch.xml)
   roslaunch_add_file_check(launch/includes/auxiliary.segbot_v1.no_sensors.launch.xml)
-  roslaunch_add_file_check(launch/segway_base.launch)
+  roslaunch_add_file_check(launch/includes/auxiliary.segbot_v2.launch.xml)
+  #roslaunch_add_file_check(launch/includes/auxiliary.segbot_v3.launch.xml)
```
Comment (piyushk) [inline segbot_bringup/CMakeLists.txt]: you could probably add this in. This launch file doesn't have a dependency on the missing v3 packages.

**Comment by jack-oquin** [comment]
xref: utexas-bwi/bwi#38

**Comment by piyushk** [comment]
looks really good @jack-oquin! Thanks for pushing it through. I had one minor comment.

File: segbot_bringup/CMakeLists.txt (hunk 1 above)
Comment (jack-oquin) [inline segbot_bringup/CMakeLists.txt]: It's still there, I just sorted it earlier in the list (line 33), with the other ``launch/`` folder contents.

**Comment by jack-oquin** [comment]
I don't think this will break anything, and it fixes some test failures.

Since enabling Travis for this repository causes errors for any other merges, I guess I'll go ahead an merge this now.

**Pull request merged**